### ISSUE TITLE
Music items are not deleted correctly

### ISSUE THREAD

**Issue by croneter** [croneter/PlexKodiConnect#336]

**Comment by croneter** [comment]
I have not been able to repro this. Anyone able to share logs and/or instructions?

**Comment by croneter** [comment]
Closing - if anyone runs into this, please provide some logs ;-)

**Comment by nerthazrim** [comment]
@croneter I confirm this issue.

For instance, when an album is miss-scanned by Plex and sync on Kodi, if I update the album in Plex, even with a PKC database reset, the audiotheque is not updated. Therefore, the music DB is not properly reset on Kodi.

What logs would you need exactly?

**Comment by croneter** [comment]
Ough... I need to reproduce the issue most of all. Will let you know if I need logs from you

**Comment by croneter** [comment]
What exactly did you change on the Plex Media Server side that was not synced over to Kodi? Can you reproduce this?

**Comment by nerthazrim** [comment]
The process that made me discover this was the following:

- I'm putting a FLAC "Deluxe" music album, splitted in 2 discs on my NAS.
- My PMS sees it, scans it but messes the scraping - some tracks are left alone in random albums, multiple albums with almost the same names are created, that kind of mess.
- This mess was sync in Kodi with PMS.
- When I went in the PMS library to clean the mess, merge the albums together, bring the tracks all in the same album, nothing is changed on Kodi's side, even with a "Local DB recuperation".

The other time where I encountered this issue is when the PMS scrapping gave the wrong name to an album. I manually re-scrapped the album and took the right one, PMS downloaded the metadata and information but nothing has been changed on Kodi's side, even with a "Local DB recuperation".
